RFCs/FS-1141-Allow-access-modifiers-to-auto-properties-getters-and-setters.md

```diff
@@ -0,0 +1,124 @@
+# F# RFC FS-1141 - Allow access modifiers to auto properties getters and setters
+
+The design suggestion [Allow access modifies to auto properties getters and setters](https://github.com/fsharp/fslang-suggestions/issues/430) has been marked "approved in principle".
+
+This RFC covers the detailed proposal for this suggestion.
+
+- [x] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/430)
+- [x] Approved in principle
+- [x] [Implementation](https://github.com/dotnet/fsharp/pull/16687)
+- [ ] Design Review Meeting(s) with @dsyme and others invitees
+<!-- - [Discussion](https://github.com/fsharp/fslang-design/discussions/FILL-ME-IN) -->
+
+# Summary
+
+Allow access modifiers to auto properties getters and setters.
+
+# Motivation
+
+To simplify the syntax of defining a private-settable-read-only properties.
+
+# Detailed design
```

Comment:
I don't think this is detailed enough. What does the new syntax compile down into? Which access modifiers are allowed? What can be mixed or not mixed? etc.

Reply:
updated